CHANGELOG.md

```diff
@@ -1,3 +1,143 @@
+# 84.0.0
+
+In this release we corrected a text field accesssibility behavior added ripple to chips and added
+additional delegate calls to post bottom sheet's scrolling.
```

Comment:
> In this release we corrected a TextFields accessibility behavior, added Ripple to Chips, and added
additional delegate calls to BottomSheet.
    
Components are always camel-cased to match their folder name.

Reply:
applied and then copyedited to remove "we"